project/runner.go

```diff
@@ -86,3 +103,208 @@ func (runner *StandardRunner) Run(ctx context.Context, r *Rule, opts RunOpts) (C
 	}
 	return OK, nil
 }
+
+// Runs commands in bash
+func (runner *StandardRunner) execRunCommand(
+	ctx context.Context,
+	r *Rule,
+	runOpts RunOpts,
+	env map[string]string,
+	cmd *Command,
+	cmdIndex int,
+) error {
+
+	execOpts := ExecOpts{
+		Command:          cmd.Argument,
+		WorkingDirectory: r.Component().Directory(),
+		Env:              flattenEnvironment(env),
+		Stdout:           runOpts.Output,
+		Stderr:           runOpts.Output,
+		Image:            r.Image(),
+		Debug:            runOpts.Debug,
+		Cmdout:           runOpts.DebugOutput,
+		Name:             fmt.Sprintf("%s.%d", r.NodeID(), cmdIndex),
+	}
+	if err := runOpts.Executor.Execute(ctx, execOpts); err != nil {
+		return fmt.Errorf("Exec error %s: %s", r.NodeID(), err)
+	}
+	return nil
+}
+
+// Creates a zip file with the specified contents. By default, the options
+// `-qr` are used: quiet and recursive. The `cd` attribute may be used to
+// change into the specified directory before running the command.
+func (runner *StandardRunner) execZipCommand(
+	ctx context.Context,
+	r *Rule,
+	runOpts RunOpts,
+	env map[string]string,
+	cmd *Command,
+	cmdIndex int,
+) error {
+
+	opts := getCommandAttr(cmd, "options", "-qr")
```

Comment:
Would you consider making this `-qrFS`? We've run into an issue where, by default, zip will overwrite existing files and append new ones, but leave old files in-place. `FS` will overwrite if the file size / mod time is different as well as delete files that no longer exist.

Reply:
Done!